src/concrete/ml/deployment/fhe_client_server.py

```diff
@@ -25,6 +26,27 @@
     from importlib_metadata import version
 
 
+class Mode(Enum):
```

Comment:
imo `Mode` is not explicit enough, even if it's defined here. Since we already have `FheMode`, maybe `ServerMode`, `RunMode`or `EncryptedMode`  ?

Reply:
`ModelMode` also could work

Reply:
DeploymentMode ?

Reply:
let's go with `FunctionToDeploy` = Training/Inference